ipaserver/install/ipa_migrate.py

```diff
@@ -0,0 +1,2028 @@
+                                 'everything will be migrated including the '
+                                 'current user sids and DNA ranges.  In '
+                                 '"stage-mode" sids, dna ranges, etc are '
+                                 'not migrated')
+        parser.add_argument('hostname',
+                            help='The FQDN hostname of the remote IPA '
+                                 'server to migrate to this local IPA '
+                                 'instance')
+        parser.add_argument('-v', '--verbose', help='Verbose output',
+                            action='store_true', default=False, dest='verbose')
+        parser.add_argument('-D', '--bind-dn',
+                            help='The Bind DN to authenticate to the remote '
+                                 f'server.  The default is "{BIND_DN}"',
+                            default=BIND_DN)
+        parser.add_argument('-w', '--bind-pw',
+                            help='Password for the Bind DN.  If a password '
+                                 'is not provided then the user will be '
+                                 'prompted to enter it',
+                            default=None)
+        parser.add_argument('-j', '--bind-pw-file',
+                            help='A text file containing the clear text '
+                                 'password for the Bind DN', default=None)
+        parser.add_argument('-Z', '--cacertfile',
+                            help='File containing a CA Certificate that the '
+                                 'remote server trusts',
+                            default=None)
+        parser.add_argument('-s', '--subtree', action='append', default=[],
+                            help='Adds an additional custom database '
+                                 'subtree to include in the migration.')
+        parser.add_argument('-l', '--log-file',
+                            help='The log file for recording the migration '
+                                 f'effort. The default is "{LOG_FILE_NAME}"',
+                            default=LOG_FILE_NAME)
+        """
+        parser.add_argument('-u', '--conflict-ldif-file',  # TODO needed?
+                            help='An LDIF file containing conflict entries, '
+                                 'or entries that need special attention '
+                                 'before they can be migrated. The default '
+                                 f'file is "{CONFLICT_FILE_NAME}"',
+                            default=CONFLICT_FILE_NAME)
+        """
+        parser.add_argument('-S', '--skip-schema',
+                            help='Do not migrate schema',
+                            action='store_true', default=False)
+        parser.add_argument('-C', '--skip-config',
+                            help='Do not migrate configuration',
+                            action='store_true', default=False)
```

Comment:
This should be clarified to match the man page to include "(dse.ldif/cn=config)"